tools/ep_kernels/install_python_libraries.sh

```diff
@@ -47,9 +47,25 @@ export NVSHMEM_IBRC_SUPPORT=1
 # remove MPI dependency
 export NVSHMEM_BUILD_TESTS=0
 export NVSHMEM_BUILD_EXAMPLES=0
-export NVSHMEM_MPI_SUPPORT=0
+export NVSHMEM_MPI_SUPPORT=1
+export NVSHMEM_BUILD_HYDRA_LAUNCHER=1
+export NVSHMEM_BUILD_TXZ_PACKAGE=0
+export CMAKE_CUDA_ARCHITECTURES=90a
+
+cmake -S . -B $WORKSPACE/nvshmem_build/ \
+-DCMAKE_INSTALL_PREFIX=$WORKSPACE/nvshmem_install \
+-DCMAKE_CUDA_ARCHITECTURES="90a" \
+-DNVSHMEM_MPI_SUPPORT=0            \
+-DNVSHMEM_PMIX_SUPPORT=0           \
+-DNVSHMEM_IBRC_SUPPORT=1           \
+-DNVSHMEM_IBGDA_SUPPORT=1          \
+-DNVSHMEM_IBDEVX_SUPPORT=1         \
+-DNVSHMEM_USE_GDRCOPY=1            \
+-DNVSHMEM_BUILD_TESTS=0            \
+-DNVSHMEM_BUILD_EXAMPLES=0         \
+-DNVSHMEM_LIBFABRIC_SUPPORT=0      \
+-DGDRCOPY_HOME=${GDRCOPY_HOME}    
```

Comment:
Can the vLLM team provide an official Dockerfile similar to [https://github.com/sgl-project/sglang/blob/main/docker/Dockerfile.deepep](https://github.com/sgl-project/sglang/blob/main/docker/Dockerfile.deepep), to make it easier for end user to use?